lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -60,157 +69,208 @@ WasmBinaryReader::WasmBinaryReader(PageAllocator * alloc, byte* source, size_t l
     ResetModuleData();
 }
 
+void WasmBinaryReader::InitializeReader()
+{
+    ModuleHeader();
+#if DBG
+    if (m_trace)
+    {
+        byte* startModule = m_pc;
+
+        bool doRead = true;
+        SectionCode prevSect = bSectInvalid;
+        while (doRead)
+        {
+            SectionHeader secHeader = ReadSectionHeader();
+            if (secHeader.code <= prevSect)
+            {
+                TRACE_WASM_DECODER(L"Unknown section order");
+            }
+            prevSect = secHeader.code;
+            // skip the section
+            m_pc = secHeader.end;
+            doRead = !EndOfModule() && secHeader.code != bSectEnd;
+        }
+        m_pc = startModule;
+    }
+#endif
+}
+
+void
+WasmBinaryReader::ThrowDecodingError(const wchar_t* msg, ...)
```

Comment:
I think I just moved this function around by mistake. Essentially I added the var args.